arch/arm/include/aarch32/cortex_r/tcm.h

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Please add this file to `arch/arm/include/aarch32/cortex_a_r` directory (`cortex_r` directory will be removed in future releases).

Reply:
done